### PR TITLE
Add nuts and bolts

### PR DESCRIPTION
**Pull request by Drew4484**
#### Summary
Content "Adds nuts and bolts."

#### Purpose of change
Adding nuts and bolts (single item because if I make players find a matching nut for each bolt, someone will DM me a picture of my front door). These can be used for construction but more importantly, can be used in place of some of the existing soldering and welding.

#### Describe the solution
Adds screw as an ammo type.

#### Describe alternatives you've considered
None.

#### Testing
Loaded game, no errors when spawning nuts and bolts.

#### Additional context
None.

### CODE REVIEW

File: tools/json_tools/generic_guns_validator.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
     'm235',  # Rocket
     'metal_rail',
     'nail',
+	'screw',
```
Comment (Maleclypse) [inline tools/json_tools/generic_guns_validator.py:29]: Linting error here it looks like

Comment (kevingranade) [inline tools/json_tools/generic_guns_validator.py:29]:
```suggestion
    'nail',
    'screw',
```

**Comment by anoobindisguise** [comment]
Why not turn nails into catchall "fasteners"? Nails and screws are similar enough that they're going to be used interchangeably in nearly anything, the only difference being one needs a screwdriver/drill and the other needs a hammer. Nail projects could be done with drills/screwdriveres or hammers. It's just a LOT of JSON work to get it all nice and differentiated!

**Comment by haveric** [comment]
> Why not turn nails into catchall "fasteners"?

I would have to agree with this, especially since we already group pipe fittings as "end caps, pipe junctions, and similar items". Fasteners could work the same way as "nails, screws, bolts, and similar items".

**Comment by Drew4484** [comment]
> > Adding a simple generic screw adds some options for construction and crafting.
> 
> Do you have anything in particular in mind here?  Right now nails are used for wood construction, and welding is used for metal.
> 
> We could have nails or wood screws for wood, but even there I'm not seeing much in the way of differentiation between the two.
> We could also have nuts and bolts for metal fastening, and metal screws and threaded parts (yikes) for fine metal fastening.
> 
> I could definitely be missing something though.

Screws/nuts and bolts give a non-welding option for metal for certain applications and with the right tools. Obtaining any of the above should be easy as screws are often the least reusable fastener. It also boosts the value of power drills/bolt drivers.

**Comment by kevingranade** [comment]
Ok so this isn't wood screws, it's nuts and bolts?  I agree that has a definite use case, but your description is based on wood screws.

**Pull request merged**